docs/participate/lite-phase.md

```diff
@@ -4,43 +4,49 @@ sidebar_position: 0
-For more details, please visit our blog post: [First Data Infrastructure Mint on Polygon](https://blog.powerloom.io/first-data-infrastructure-mint-on-polygon-launching-over-10-000-data-nodes/).
+### Key Milestones in Our Node Mint Journey:
+- ✅ Pre-Mint Kickoff: January 25th, 2024
+- ✅ Pre-Mint Closure: February 4th, 2024
+- ✅ Pre-Mint Selection: February 6th, 2024
+- ✅ Early Mint (Exclusive): February 7th, 2024 
+- ⏳ Public Mint Opens: February 9th, 2024 (ongoing)
 
-:::info
-Important Announcement: Our Testnet is gearing up for launch shortly. In the interim, we invite developers to experiment with running our Snapshotter node.
 
-While currently operating the node does not yield rewards, this is an excellent opportunity to simulate and test the Snapshotter node, providing valuable insights into its functionality and operation.
-:::
+For detailed information, please refer to our Snapshotter Node Mint documentation: [Snapshotter Node Mint Documentation](https://powerloom.network/mint-docs/)
+
+For additional insights, visit our blog post: [First Data Infrastructure Mint on Polygon](https://blog.powerloom.io/first-data-infrastructure-mint-on-polygon-launching-over-10-000-data-nodes/).
 
-## Introducing Snapshotter Lite Nodes
+:::info Important Announcement
+ As we prepare for our Testnet launch, we invite developers to explore and experiment with operating our Snapshotter node. Currently, operating the node does not provide rewards, but it offers a valuable opportunity to simulate, test, and gain insights into the Snapshotter node's functionality and operation.
+:::
 
-After acquiring node slots, you can operate our Snapshotter Lite Node. This version is more streamlined than the full Snapshotter Node, requiring only a Python environment and considerably fewer resources. It's ideal for simpler scenarios that don't need data aggregation.
+## What Are Snapshotter Lite Nodes?
 
-## How Snapshotter Lite Node Functions within the PowerLoom Ecosystem
+Once you have acquired your node slots, you can begin operating our Snapshotter Lite Node. This streamlined version of the full Snapshotter Node requires only a Python environment and significantly fewer resources, making it ideal for less complex scenarios that do not require data aggregation.
 
-The architecture of the Snapshotter Lite Node primarily differs from the full node implementation mainly because it reduces the overhead of messaging queues and streamlines other complex components. This is essential for contributing to data markets, beginning with the UniswapV2 Data Markets.
+## Role of the Snapshotter Lite Node in the PowerLoom Ecosystem
+The Snapshotter Lite Node's architecture is distinct from the full node version, in that messaging queue overheads are reduced and complex components have been simplified. This streamlined design is crucial for contributing to data markets, starting with the [UniswapV2 Data Markets](https://uniswapv2.powerloom.io).
```

Comment:
crucial to enhancing the security of data markets by expanding the network of peers participating in them, starting with...